container/layer.bzl

```diff
@@ -194,6 +206,8 @@ _layer_attrs = dict({
     # Implicit/Undocumented dependencies.
     "empty_files": attr.string_list(),
     "empty_dirs": attr.string_list(),
+    "architecture": attr.string(default="amd64", mandatory = False),
```

Comment:
does not seem like you are using this architecture attr anywhere in the impl?

Reply:
Plumbed it all the way through now to create_image_config.py.